src/GuppyDataExplorer/ExplorerTable/index.jsx

```diff
@@ -76,7 +76,8 @@ class ExplorerTable extends React.Component {
           const nestedChildFieldName = fieldStringsArray.slice(1, fieldStringsArray.length).join('.');
           // some logic to handle depends on wether the child field in raw data is an array or not
           if (_.isArray(row.value)) {
-            valueStr = row.value.map((x) => _.get(x, nestedChildFieldName)).join(', ');
+            // replace empty nested values with null so we can still join cleanly and display 1 value per nested row
+            valueStr = row.value.map((x) => _.get(x, nestedChildFieldName) || 'null').join(', ');
```

Comment:
Is it possible for `_.get(x, nestedChildFieldName)` to return a value other than string? If so, then falsy values like 0 for numeric types or even a "false" boolean will show up as 'null'.